server/services/homekit/lib/buildService.js

```diff
@@ -68,6 +68,15 @@ function buildService(device, features, categoryMapping) {
         });
         break;
       }
+      case `${DEVICE_FEATURE_CATEGORIES.MOTION_SENSOR}:${DEVICE_FEATURE_TYPES.SENSOR.BINARY}`: {
```

Comment:
You can use same case as `${DEVICE_FEATURE_CATEGORIES.LEAK_SENSOR}:${DEVICE_FEATURE_TYPES.SENSOR.BINARY}` or `${DEVICE_FEATURE_CATEGORIES.OPENING_SENSOR}:${DEVICE_FEATURE_TYPES.SENSOR.BINARY}` depending on whether or not the state is to be inverted for Homekit